pkgs/by-name/ch/chartdb/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_24,
+}:
+let
+  name = "chartdb";
+  version = "1.19.0"; # https://github.com/chartdb/chartdb/releases
+in
+buildNpmPackage {
+  pname = name;
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = name;
+    repo = name;
```

- instance of #277994
```suggestion
    owner = "chartdb";
    repo = "chartdb";
```